### DISCUSSION TITLE
Can User "Generate PDF" without being in wordpress dashboard?

### DISCUSSION THREAD

**Discussion by Area48Webmaster** [code4recovery/12-step-meeting-list#650] [Q&A] [answered]
Hi all, always grateful for what the developers do and have done for our community.

My question is if there is a way for a user to create a PDF to print from what they have filtered through on the website. I see the Generate PDF button on the dashboard of wordpress, but what we're really trying to find is a way for the AA meber who logs onto our website, or even a treatment professional, to be able to print a meeting list from what they are searching and have filtered on the public meeting page.

For example. A newcomer goes to the library for internet and searches for meetings on Tuesday in District 1. Can he then print out that list of meetings from what he has searched?

Really hoping this is possible! Thank you again.

**Comment by tech2serve** [comment, marked as answer]
Dang. Every now and again, we completely miss a posted question. I apologize profusely for this, and promise we'll continue to strive to respond to user questions in a timely manner.

The short answer is we don't directly support what you're requesting, and I don't believe we would consider it since the OS already provides a pretty simple solution.

The simplest solution is for the end user to make a screen cap of the filtered meeting list, and then print that to a PDF.  Make sense?
